includes/privacy-functions.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Privacy Functions
+ *
+ * @package     EDD
+ * @subpackage  Functions
+ * @copyright   Copyright (c) 2018, Easy Digital Downloads, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       2.9.x
+ */
+
+// Exit if accessed directly
+if ( ! defined( 'ABSPATH' ) ) exit;
+
+
+function edd_register_privacy_policy_template() {
+
+	if ( ! function_exists( 'wp_add_privacy_policy_content' ) ) {
+		return;
+	}
+
+	$content = wp_kses_post( apply_filters( 'edd_privacy_policy_content', __( '
+We collect information about you during the checkout process on our store. This information may include, but is not limited to, your name, billing address, shipping address, email address, phone number, credit card/payment details and any other details that might be requested from you for the purpose of processing your orders.
+Handling this data also allows us to:
+- Send you important account/order/service information.
+- Respond to your queries, refund requests, or complaints.
+- Process payments and to prevent fraudulent transactions. We do this on the basis of our legitimate business interests.
+- Set up and administer your account, provide technical and/or customer support, and to verify your identity.
+' ) ) );
+
+	$content .= "\n\n";
+
+	$additional_collection = array(
+		__( 'Location and traffic data (including IP address and browser type) if you place an order, or if we need to estimate taxes and shipping costs based on your location.', 'easy-digital-downloads' ),
+		__( 'Product pages visited and content viewed whist your session is active.', 'easy-digital-downloads' ),
```

Comment:
`whist` -> `whilst`? Why not `while`? Copy pasta or tailoring it to the region?

Reply:
@SDavisMedia I'm using one that was built for a simliar product as a template. the `whilst` was probably something their legal team mocked up. I'll fix it to `while`.